test/libsolidity/util/TestFileParser.cpp

```diff
@@ -75,43 +75,55 @@ vector<dev::solidity::test::FunctionCall> TestFileParser::parseFunctionCalls(siz
 
 				try
 				{
-					tie(call.signature, call.useCallWithoutSignature) = parseFunctionSignature();
```

Comment:
This is mostly whitespace changes, so with ``Hide whitespace changes`` set, the change is rather trivial.